birdy/cli/base.py

```diff
@@ -23,14 +23,16 @@ class BirdyCLI(click.MultiCommand):
     :param url: URL of the Web Processing Service.
     :param caps_xml: A WPS GetCapabilities response for testing.
     :param desc_xml: A WPS DescribeProcess response with "identifier=all" for testing.
+    :param language: Language parameter for WPS GetCapabilities and DescribeProcess responses.
     """
-    def __init__(self, name=None, url=None, caps_xml=None, desc_xml=None, **attrs):
+    def __init__(self, name=None, url=None, caps_xml=None, desc_xml=None, language=None, **attrs):
```

Comment:
hmm.. I added it mainly so it could be passed on for self.wps = WebProcessingService(...), so an instance of the BirdyCLI class could be initiated with a language, but it might be a lack of comprehension of how click.MultiCommand works and is used.

Should I remove the language parameter from the __init__ and WebProcessingService instantiation and only add it as a command?

Reply:
I'm not familiar enough with click either... I would need to check into this more.

If you want, you can focus this PR on adding language support for the client only, or find how to add it as a parameter to the cli also.

Reply:
Alright, well I don't think adding it to the cli is bad per say, as it still is a class that has a wps as a parameter, which can receive a language parameter. It might not be a best practice for click though...

I have time to dig a bit more into click. If it turns out to be a time sink or I get submerged by other assignments, I'll revert what I changed to BirdyCli and we'll save it for a future PR.